### DISCUSSION TITLE
Authentication email appears collapsed

### DISCUSSION THREAD

**Discussion by Ddupasquier** [supabase/supabase#12338] [Questions]
When sending a user a magic link for authentication, the email always appears collapsed.
![image](https://user-images.githubusercontent.com/89223122/218219929-580ce05b-220b-4452-b278-3ecf688a47af.png)

Is this a default for gmail for certain kinds of emails? Or is this something I can adjust from the supabase UI?

**Comment by silentworks** [comment]
This is a gmail thing for repeat emails to the same user from the same domain with the same content.

**Reply by Ddupasquier** [reply]
Ahhh, okay. Unfortunate xD

Thanks for the reply!
